packages/pagination/Pagination.tsx

```diff
@@ -0,0 +1,276 @@
+import * as React from "react";
+import Delegate from "react-delegate-component";
+import { useId } from "react-id-generator";
+import { cx } from "emotion";
+import { Flex, FlexItem } from "../styleUtils/layout";
+import { Icon } from "../icon";
+import { ResetButton } from "../button";
+import { ResetLink } from "../link";
+import { SelectInput } from "../selectInput";
+import { SystemIcons } from "../icons/dist/system-icons-enum";
+import { TextInput } from "../textInput";
+import { themeBgDisabled } from "../design-tokens/build/js/designTokens";
+import { ExpandedLinkProps } from "../link/types";
+import { defaultInputWidth } from "./style";
+import { border, display, margin, padding } from "../shared/styles/styleUtils";
+
+interface PaginationProps {
+  /** Prop used to control which page the user is currently navigated to */
+  activePage?: number;
+  /** The default page the user starts on before the user navigates using the text input or the prev/next buttons. For use when the component is uncontrolled. */
```

Comment:
just for my gut feeling: do we use (a lot of) uncontrolled components in DKP-UIs?

Reply:
I think we mostly use controlled components